cross/pillow/Makefile

```diff
@@ -1,5 +1,5 @@
 PKG_NAME = Pillow
-PKG_VERS = 8.2.0
+PKG_VERS = 8.4.0
```

Comment:
unfortunately this breaks homeassistant (#4919) as it requires pillow==8.2.0.
IMHO we should include as less wheels as possible in the python* packages.
Or we could separate cross/pillow_py38 to version 8.2.0 for python38.

Reply:
fixed for homeassistant in #4919 by use of dedicated version in `cross/homeassistant.pillow`.